sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -712,12 +712,15 @@ typedef enum
 {
   AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_GET
   = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_GET, /**< A response from a properties "GET" request. */
-  AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_DESIRED_PROPERTIES
+  AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_WRITEABLE
   = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES, /**< A "PATCH" response with a payload
-                                                                containing desired properties. */
-  AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_REPORTED_PROPERTIES
-  = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES, /**< A response with the result of the
-                                                                 earlier reported properties. */
+                                                                containing updated writeable
+                                                                properties for the device to
+                                                                process. */
+  AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_ACKNOWLEDGEMENT
+  = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES, /**< A response acknowledging the
```

Comment:
ACKNOWLEDGEMENT could also refer to a GET request response, couldn't it?  Can we add PATCH to this?  like, PATCH_ACKNOWLDGEMENT? or PATCH_RESPONSE?

Reply:
I think this would be an acknowledgement of either a writeable property originating from the device or the service is pushing down the property, the device acknowledges that it updated, and then service then acknowledges it received the ack. The get response would still be `_GET`

Reply:
Related to the "RESPONSE" => "MESSAGE" change, on GET response I'm changing it to be `AZ_IOT_HUB_CLIENT_PROPERTIES_MESSAGE_TYPE_GET_RESPONSE.`  So it maintains the "MESSAGE" infix but also shows that the GET here is *NOT* the server trying to get anything from the device, but a response from the device init'd get.

I still tend to like the ACK for a property send.  We'll see how it looks in toto when I get next round of this out.